inc/alert.class.php

```diff
@@ -214,7 +214,7 @@ public static function findAllToNotify($params = []) {
       $login_show_hidden_sql = " `$utable`.`id` IS NULL ";
       $entity_sql            = "";
       $show_helpdesk_sql     = '';
-      if (isset($_SESSION['glpiID'])) {
+      if (isset($_SESSION['glpiID']) && isset($_SESSION['glpiactiveprofile']['id'])) {
```

Comment:
Problem is is that, in this specific case, it will just remove some filtering, and may expose alerts that are not meant to be displayed to this user.
I guess in this case we should remove the `` `$ttable`.`items_id` = ".$_SESSION['glpiactiveprofile']['id']." OR `` part of the query to get elements that matches only `` `$ttable`.`itemtype` = 'Profile' AND `$ttable`.`items_id` = -1 ``

Reply:
I'm not very sure about this part of code, but  I think that if "glpiID" && "glpiactiveprofile:id" are not set, then I guess we are on GLPI login page, and therefore the filter is done further down in the WHERE with **$login_sql** variable. 

But maybe I was all wrong, in this case: can you push your commit directly?